gpu-direct/ubuntu/entrypoint.sh

```diff
@@ -3,6 +3,14 @@ MOFED=/run/mellanox/drivers
 NVIDIA=/run/nvidia/drivers
 KERNEL_VERSION=$(uname -r)
 
+function set_driver_readiness() {
+    touch /driver-ready
```

Comment:
can we have /.driver-ready?

Reply:
This is later being read by startupProbe in K8s , do we want this file to be hidden ?

also i was thinking in putting it in its own folder in the unlikely event more files would be needed
or is that just overkill for what we are trying to achieve?

Reply:
why we need user to see this files  if be used internally?  
regrading the folder you call I don't mind

Reply:
> why we need user to see this files 

Technically no one should really exec into this container unless he/she is debugging.
the consumer of the file is k8s startupProbe or other orchestrators that want a way to ensure the driver was loaded before proceeding to other tasks.
Im also thinking we may want to document this.

>  be used internally?

It is not used internally, its being used by other project.

Reply:
I understand that, but  it is still implementation internal just like a lock file.  You create empty file just that you can synchronized between jobs/processes whatever ...
 I don't understand why you against it?

Reply:
i guess its just semantics,
as a dotfile is hidden, so i think it hints that no one apart of the creator of the file should be aware of it.
maybe its my misconception of the purpose of a dofile.

ill prefix the file with a `.`